src/main/scala/io/getclump/package.scala

```diff
@@ -0,0 +1,15 @@
+package io
+
+package object getclump {
+
+  private[getclump] implicit val executionContext = scala.concurrent.ExecutionContext.global
```

Comment:
Ya I don't know how to get around execution context without adding it to every method. 95% of people will be fine with the global EC, my use case is pretty special so I can just maintain that one line patch.